files/ubuntu-adfc.css

```diff
@@ -0,0 +1,2011 @@
+/* This stylesheet is generated, DO NOT EDIT */
+/* Copyright 2009, 2015 Red Hat, Inc.
+ *
+ * Portions adapted from Mx's data/style/default.css
+ *   Copyright 2009 Intel Corporation
+ *
```

Comment:
Was genau wurde in dieser Datei durch uns geändert?

Reply:
in Zeile 1814 (Block "#lockDialogGroup {...") wurde die Verlinkung zum Hintergrundbild hinzugenommen.